pyfr/writers/base.py

```diff
@@ -33,4 +33,8 @@ def __init__(self, args):
                                (args.solnf, args.meshf))
 
         # Load config file
-        self.cfg = Inifile(self.soln['config'].item().decode())
+        self.cfg = Inifile(
+                (self.soln['config'].item()
+                    if type(self.soln['config'].item()) is str
+                    else self.soln['config'].item().decode())
+                          )
```

Comment:
Split out to simplify.  Grab `self.soln['config'].item()` into a variable and go from there.  Use of `type` for instance checking; please see Python documentation.